packages/terra-functional-testing/src/services/wdio-visual-regression-service/VisualRegressionLauncher.js

```diff
@@ -1,12 +1,10 @@
 /* global browser */
-import _ from 'lodash';
-import { parse as parsePlatform } from 'platform';
-
-import { LocalCompare } from './compare';
-import makeElementScreenshot from './modules/makeElementScreenshot';
-
-import getUserAgent from './scripts/getUserAgent';
-import getTerraFormFactor from './modules/getTerraFormFactor';
+const _ = require('lodash');
```

Comment:
instead of pulling in all of lodash, could we just pull in what we need?

Reply:
Updated to use only needed lodash modules: https://github.com/cerner/terra-toolkit/pull/545/commits/90db6b082ead33e7514b4178e5d509cf8f1cf39c